### ISSUE TITLE
posix_net_conf.sh with bonded nic fails with error in 1.6.2

### ISSUE THREAD

**Issue by kthommandra** [scylladb/scylla#2269]
*Installation details*
Scylla version (or git commit hash): 1.6.2-20170310.d41ba3b
Cluster size:
OS (RHEL/CentOS/Ubuntu/AWS AMI): CentOS 7.2

```
[root@mwXX ~]# scylla_setup --no-node-exporter --no-ntp-setup --no-selinux-setup --no-raid-setup --no-enable-service --no-bootparam-setup --setup-nic --nic bond0
Skip any of the following steps by answering 'no'
This is a supported kernel version.
kernel.core_pattern = |/usr/lib/systemd/systemd-coredump %p %u %g %s %t %e
Setting parameters on /etc/sysconfig/scylla-server
**/usr/lib/scylla/posix_net_conf.sh: line 349: found_mq: command not found**
```

The fix seems to be to have the following at line 349

```
if [[ ! -z $found_mq ]]; then
```

**Comment by glommer** [comment]
@vladzcloudius please take a look at this.

1.6 and 1.7 still use the legacy posix_net_conf.sh, so we'll need a separate fix for those.

**Comment by vladzcloudius** [comment]
@slivne The patches for 1.6 and 1.7 are on the list.

**Comment by slivne** [comment]
@vladzcloudius - can we close this was everything merged

**Comment by vladzcloudius** [comment]
@slivne As far as I can tell it was only merged into the 1.6 branch. A patch for 1.7 is still *not* merged.

**Comment by vladzcloudius** [comment]
@slivne I've just noticed that it was fixed the different way in 1.6 and 1.7 - they merged the transition of `posix_net_conf.sh` to `perftune.py`, which automatically fixes this issue. 
So, yes, the issue may ne safely closed.
